tests/PhpSpreadsheetTests/Cell/CellTest.php

```diff
@@ -130,10 +140,9 @@ public function testNoChangeToActiveSheet(): void
         $sheet1->setCellValue('D1', 124);
         $sheet3->setCellValue('A1', "='Sheet 1'!C1+'Sheet 1'!D1");
         $sheet1->setCellValue('A1', "='Sheet 3'!A1");
-        $cell = 'A1';
         $spreadsheet->setActiveSheetIndex(0);
         self::assertEquals(0, $spreadsheet->getActiveSheetIndex());
```

Comment:
As long as you are cleaning up this test member anyhow, would you mind changing all the `assertEquals` to `assertSame` where possible (which I'm guessing is all of them). It's a stronger test.